lib/container_orchestrator.rb

```diff
@@ -58,7 +58,21 @@ def delete_secret(name)
     raise unless e.message =~ /not found/
   end
 
+  def get_pods
+    # TODO: we should set a label for workers we want to monitor so we don't accidentally kill memcached/postgresql pod
+    # :label_selector => "app=manageiq,worker=true"
```

Comment:
We might be able to use the fact that the orchestrator is set as the owner, and then only query things owned by the orchestrator.